02-peer-protocol.md

```diff
@@ -138,9 +177,9 @@ The sending node:
   - MUST set `first_per_commitment_point` to the per-commitment point to be used for the initial commitment transaction, derived as specified in [BOLT #3](03-transactions.md#per-commitment-secret-requirements).
   - MUST set undefined bits in `channel_flags` to 0.
   - if both nodes advertised the `option_upfront_shutdown_script` feature:
-	- MUST include either a valid `shutdown_scriptpubkey` as required by `shutdown` `scriptpubkey`, or a zero-length `shutdown_scriptpubkey`.
+  - MUST include either a valid `shutdown_scriptpubkey` as required by `shutdown` `scriptpubkey`, or a zero-length `shutdown_scriptpubkey`.
   - otherwise:
-	- MAY include a`shutdown_scriptpubkey`.
+  - MAY include a`shutdown_scriptpubkey`.
 
```

Comment:
Previous indentation on these was correct: they are dependent on the line above.

Reply:
Woop! That was a cut-and-paste problem as I incorporated other changes that had appeared since I started editing.

Reply:
You didn't fix the one above, but I'll do that now...  Also, indentation is two spaces.